### PR TITLE
Fix the property `disturbance`

### PR DESCRIPTION
**Pull request by junrongYu**
Fixes: #71

### CODE REVIEW

**Comment by junrongYu** [comment]
The source, definition, and feature type need to be confirmed.

**Comment by edmondchuc** [comment]
> The source, definition, and feature type need to be confirmed.

I'm guessing this is not ready to be reviewed yet then?

Also, shouldn't there be 2 disturbance properties deleted, not just one? The original issue states there are 3 in total.

**Comment by junrongYu** [comment]
> > The source, definition, and feature type need to be confirmed.
> 
> I'm guessing this is not ready to be reviewed yet then?
> 
> Also, shouldn't there be 2 disturbance properties deleted, not just one? The original issue states there are 3 in total.

Hi @edmondchuc , those 2 `disturbance` in the issue are actually the same property with different definitions, so I was asking for the correct definition then. @arunherb said he'll review definitions later.

**Comment by junrongYu** [comment]
The feature type is `land surface disturbance`.

File: vocab_files/observable_property_concepts/disturbance.ttl [hunk 1]
```diff
@@ -7,21 +7,20 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
...
+    dcterms:source "Ecological Field Monitoring protocols - Soil module, Targeted Surveys module draft v0.1, 30/11/2021" ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
+    skos:altLabel "land surface disturbance" ;
     skos:closeMatch <http://linked.data.gov.au/def/tern-cv/bf6bf3fc-78f6-47f1-884b-4c46298f4fcd> ;
-    skos:definition
-        "Evidence of anthropogenic disturbance at a site. These include grazing,  clearing and land-use change. " ,
-        "The disturbance factor (chosen from a drop-down list) indicating the level of disturbance of the targeted ecological community in study." ;
+    skos:definition "Evidence of any disturbance at a given site. These include grazing,  clearing and land-use change. (page 128 Yellow book)" ;
+    skos:exactMatch <http://linked.data.gov.au/def/tern-cv/777773d0-cfb4-44ff-85e2-08b48e10ac52> ;
     skos:prefLabel "disturbance" ;
```
Comment (edmondchuc) [inline vocab_files/observable_property_concepts/disturbance.ttl:16]: This observable property is about the disturbance type found at a site.

```suggestion
    skos:prefLabel "disturbance type" ;
```

File: vocab_files/observable_property_concepts/disturbance.ttl [hunk 2]
```diff
@@ -7,21 +7,20 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 <https://linked.data.gov.au/def/test/dawe-cv/632e4c3c-0e73-4b3d-b1d6-7b38897da5e8>
     a skos:Concept ;
-    dcterms:source """McCallum K, Bignall J, Laws M, O’Neill S, Sparrow B. (unpublished draft) ‘Targeted Surveys module’ in ….. (eds) Ecological
-Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision
-making. Draft v 0.1 Report to DAWE. TERN, Adelaide.""" ;
+    dcterms:source "Ecological Field Monitoring protocols - Soil module, Targeted Surveys module draft v0.1, 30/11/2021" ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
+    skos:altLabel "land surface disturbance" ;
```
Comment (edmondchuc) [inline vocab_files/observable_property_concepts/disturbance.ttl:12]: I don't think having `land surface disturbance` as an alternative label makes sense. You're basically saying all instances of disturbance type is some kind of land surface disturbance, which is not true. Land surface is a specific type of survey (see Yellow Book) and disturbances can happen in many different types of surveys (e.g., vegetation, soil).

**Review by edmondchuc — CHANGES_REQUESTED** [review]

**Review by edmondchuc — APPROVED** [review]

**Pull request merged**